dockerfiles/Dockerfile-mina-rosetta

```diff
@@ -46,41 +64,14 @@ RUN curl -s https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
        postgresql-"$psql_version" \
        postgresql-client-"$psql_version"
 
-# --- Debian 11 "Bullseye" Dependencies
-RUN test "$deb_codename" = "bullseye" \
-     && apt-get update -y \
-     && apt-get install -y \
-         libjemalloc2 \
-         libprocps8 \
-         libffi7 \
-     || exit 0
-
-# --- Debian 10 "Buster" Dependencies
-RUN test "$deb_codename" = "buster" \
-     && apt-get update --quiet --yes \
-     && apt-get install --quiet --yes --no-install-recommends \
-         libjemalloc2 \
-         libprocps7 \
-         libffi6 \
-     || exit 0
-
-# --- Ubuntu 20.04 "Focal Fossa" Dependencies
-RUN test "$deb_codename" = "focal" \
-     && apt-get update --quiet --yes \
-     && apt-get install --quiet --yes --no-install-recommends \
-         libjemalloc2 \
-         libprocps8 \
-         libffi7 \
-     || exit 0
+# --- Golang install of a given GO_VERSION (add -v for spam output of each file from the go dist)
+RUN curl -s "https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz" | tar -xz -C /usr/lib/
 
 # --- Install rosetta-cli
-
 # The following commands install rosetta-cli with an updated rosetta-sdk-go dependency, which supports delegation transactions.
 # Upstream PR to rosetta-sdk-go: https://github.com/coinbase/rosetta-sdk-go/pull/457
 # They can be replaced once there is a new release of rosetta-cli containing the above change by:
-# RUN curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/${ROSETTA_CLI_VERSION}/scripts/install.sh | sh -s
-COPY --from=builder /usr/lib/go /usr/lib/go
-RUN apt-get install -y gcc
+#RUN curl -sSfL https://raw.githubusercontent.com/coinbase/rosetta-cli/${ROSETTA_CLI_VERSION}/scripts/install.sh | sh -s
```

Comment:
Why is this line commented out? If it's unnecessary, let's delete it.

Reply:
There is an explanation above to uncomment this line once coinbase fix delegation transaction support